### PR TITLE
Proposing additional samples for greater coverage of mssql database plugin

### PR DESCRIPTION
**Pull request by hashitop**

### CODE REVIEW

File: website/source/api/secret/databases/mssql.html.md [hunk 1]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+- `password` `(string: "")` - The root credential password used in the connection URL.
 
-### Sample Payload
+### MSSQL DSN
 
+The MSSQL database plugin supports connection URL in three forms
+
+- `sqlserver` `scheme` - URL format of connection string with `sqlserver` protocol prefix, for example
+
+  - sqlserver://{{username}}:{{password}}@localhost:1433
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `sqlserver://{{username}}:{{password}}@localhost:1433`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 2]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+  - sqlserver://yourserver:1433?connection+timeout=30&user+id=youruser&password=yourpassword
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `sqlserver://yourserver:1433?connection+timeout=30&user+id=youruser&password=yourpassword`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 3]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+  - sqlserver://yourserver:1433?connection+timeout=30&user+id=yourdomain\youruser&password=yourpassword
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `sqlserver://yourserver:1433?connection+timeout=30&user+id=yourdomain\youruser&password=yourpassword`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 4]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+
+- `ADO` `key/value style` - Key/Value pair format with semicolon separator, for example
+
+  - server=localhost\\SQLExpress;user id=sa;password=yourpassword;database=master;app name=MyAppName
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `server=localhost\\SQLExpress;user id=sa;password=yourpassword;database=master;app name=MyAppName`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 5]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+  - server=localhost\\SQLExpress;user id=yourdomain\youruser;password=yourpassword;database=master;app name=MyAppName
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `server=localhost\\SQLExpress;user id=yourdomain\youruser;password=yourpassword;database=master;app name=MyAppName`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 6]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+
+- `ODBC` `key/value style` - Key/Value pair format with `odbc:` prefix, for example
+  - odbc:server=localhost\\SQLExpress;user id=sa;password=yourpassword;database=master;app name=MyAppName
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `odbc:server=localhost\\SQLExpress;user id=sa;password=yourpassword;database=master;app name=MyAppName`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 7]
```diff
@@ -42,10 +42,49 @@ has a number of parameters to further configure a connection.
...
+  - odbc:server=localhost\\SQLExpress;user id=yourdomain\youruser;password=yourpassword;database=master;app name=MyAppName
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]:
```suggestion
  - `odbc:server=localhost\\SQLExpress;user id=yourdomain\youruser;password=yourpassword;database=master;app name=MyAppName`
```

File: website/source/api/secret/databases/mssql.html.md [hunk 8]
```diff
@@ -57,6 +96,42 @@ has a number of parameters to further configure a connection.
...
 ```
+- sqlserver scheme with Windows domain user
+```json
+{
+  "plugin_name": "mssql-database-plugin",
+  "allowed_roles": "readonly",
+  "connection_url": "sqlserver://localhost:1433?user+id={{username}}&password={{password}}",
+  "max_open_connections": 5,
+  "max_connection_lifetime": "5s",
+  "username": "yourdomain\youruser",
```
Comment (tyrannosaurus-becks) [inline website/source/api/secret/databases/mssql.html.md]: Coming through with an invalid character here.

**Comment by VioletHynes** [comment]
As this PR is older and cannot currently be merged due to merge conflicts, I’m going to go ahead and close it. Please feel free to open a new PR if your desired changes are still applicable!

**Pull request closed without merging**